deploy/deploy.sh

```
@@ -0,0 +1,293 @@
+    log_info "빌드가 성공했습니다."
+  else
+    log_error "빌드가 실패했습니다."
+    exit 1
+  fi
+}
+
+# 애플리케이션 실행 함수
+start_application() {
+  log_info "애플리케이션을 시작합니다..."
+
+  cd $PROJECT_PATH
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)